client-participation/js/util/facebookButton.js

```diff
@@ -59,30 +59,31 @@ function facebookLoginOkHandler(response, optionalPassword) {
 
 function fbLoginPrompt() {
   var dfd = $.Deferred();
-  FB.login(
-    function(response) {
+  FB.getLoginStatus(function(response) {
+    if (response.status !== 'connected') {
+      return FB.login(function(response) {
       if (response.authResponse) {
         return facebookLoginOkHandler(response).then(dfd.resolve, dfd.reject);
       } else {
         return dfd.reject();
       }
-    }, {
+    } , {
       return_scopes: true, // response should contain the scopes the user allowed
-      scope: [
-        // 'taggable_friends', // requires review.
-        // invitable_friends NOTE: only for games with a fb Canvas presence, so don't use this
-        'public_profile',
-        'user_location',
-        'user_friends',
-        'email'
-      ].join(',')
+      //scope: ['public_profile','user_location','user_friends','email'].join(',')
+      scope: ['public_profile','email'].join(',')
```

Comment:
```suggestion
      scope: ['public_profile', 'email', 'user_friends'].join(',')
```

User friends scope is used by math to determine which avatars to show in visualization. It shows friends if they have voted yet :)

FB login flow allows people to also decline to share this permission. We have another PR to allow server administrators to opt not to ever ask for it.